go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/imdario/mergo v0.3.8 // indirect
 	github.com/onsi/ginkgo v1.12.0 // indirect
 	github.com/onsi/gomega v1.9.0 // indirect
-	github.com/openebs/api v1.10.0-RC1.0.20200528063752-522f45d17d59
+	github.com/openebs/api v1.10.0-RC1.0.20200602151240-2b7d2bdbe1ef
```

Comment:
why not 1.10?

Reply:
not sure it is autogenerated by `go get github.com/openebs/api@master`

Reply:
that's is fine